executable/SummarizeApp.re

```diff
@@ -249,7 +250,29 @@ let _ =
     {
       let callback =
         Graphql_cohttp_lwt.make_callback(
-          _req => Context.{user: Some(mockedUser)},
+          req =>
+            Cohttp.Header.get(req.headers, "token")
```

Comment:
this looks to work ok but it's hard to plug other stuff after, we could break into parts and in the end each part would return its addition to the context
```
Lwt_main.run({
 let%lwt connection = Database.pool;
 let%lwt user = {
   Cohttp.Header.get(req.headers, "token")
   |> (
   fun 
    | Some(token) =>
        let userId = Jwt.( etc );
        let%lwt user = User.Model.findOne(~connection, ~clause=etc, ());
        Lwt.return(Some(user))
    | None => Lwt.return(None)
   )
 }
 
 Lwt.return(Context.{ user, connection });
})
```